lib/expansion_rules.rb

```diff
@@ -44,6 +44,13 @@ def details_fields(*fields)
     pages_secondary_to_step_nav: :secondary_to_step_navs
   }.freeze
 
+  HIGH_PRIORITY_DEPENDENCY_RESOLUTION = {
```

Comment:
Given that this doesn't end up on the `HIGH_QUEUE`, maybe call this just `PRIORITY_DEPENDENCY_RESOLUTION`, I think that gets the purpose across.